packages/third-party/tailwind-preset/vitest.config.ts

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    environment: 'node',
```

Comment:
You would also need a `name` for this test so that it could be run on CI.

```suggestion
    name: 'tools/tailwind-preset',
    environment: 'node',
```

This is slightly tricky since we are separating the tests using `--project` in the CI pipeline.

https://github.com/lynx-family/lynx-stack/blob/54861d0050c16625cb432e5ba8e02be3f25ed2ac/.github/workflows/test.yml#L163